client/packages/inventory/src/Stocktake/DetailView/Toolbar.tsx

```diff
@@ -66,7 +68,15 @@ export const Toolbar: FC = () => {
           {isDisabled && <InfoPanel message={infoMessage} />}
         </Grid>
 
-        <Grid item>
+        <Grid item display="flex" gap={1} justifyContent="flex-end">
+          <SearchBar
+            placeholder={t('placeholder.filter-items')}
```

Comment:
It occured to me that it might be nice if this changed from "Filter items" to "Filter lines" depending on the state of the "Group by items" toggle. However, we don't have access to that state from here, so it would be easier to do it after (or part of) #1760